.gitignore

```diff
@@ -19,8 +19,8 @@ config.h.in
 config.status
 configure
 libtool
-rauc
-rauc.map
```

Comment:
Did you run into any issue with this? Or just because this is the default build location for these files?

Nit: what is meant here is probably als more *'ignore rauc and rauc.map **only** in the top-level dir'* (as we still ignore other files in the top-level dir, too).

Reply:
Hmm, I had a rauc directory somewhere which was ignored by mistake, so I wanted to have the .gitignore more specific.